src/Avalonia.Base/AvaloniaPropertyRegistry.cs

```diff
@@ -30,6 +30,11 @@ public class AvaloniaPropertyRegistry
         public static AvaloniaPropertyRegistry Instance { get; }
             = new AvaloniaPropertyRegistry();
 
+        /// <summary>
+        /// Gets a list of all registered properties.
+        /// </summary>
+        internal IList<AvaloniaProperty> Properties => _properties;
```

Comment:
Nit: could we make this an `IReadOnlyList<AvaloniaProperty>` so that it doesn't accidentally get changed?

Reply:
Done.